test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_pattern.test

```diff
@@ -33,7 +33,7 @@ brokerStart CB
 # 01. GET /v2/entities?idPattern=E.*
 # 02. POST /v2/entities, creating E1
 # 03. GET /v2/entities?idPattern=E.*
-# 04. GET /v2/entities?idPattern=E1&count=true
+# 04. GET /v2/entities?idPattern=E1&options=count,normalized
```

Comment:
Did this not make the test fail?
Perhaps we're still supporting count=true but we should signal an error instead?